jujugui/static/gui/src/app/components/entity-content-readme/entity-content-readme.js

```diff
@@ -77,8 +77,10 @@ YUI.add('entity-content-readme', function() {
       @param {Object} data The returned data for the readme.
     */
     _getReadmeSuccess: function(data) {
-      var readme = data.target.responseText;
-      this.setState({readme: this.props.renderMarkdown(readme)});
+      if (this.isMounted()) {
```

Comment:
I'm guessing this is because sometimes the readme would come back after it had already been closed? While this is good I figure that we should probably use `componentWillUnmount` and cancel the inflight request.